File: client/src/app/models/groupe.ts
import {Horaire} from "./horaire";

export class Groupe {

  bold!: boolean;
  checked!: boolean;
  horaires!:Horaire[];
  id!:number;
  indent!:boolean;
  nom!: string;
  ordre!: number;
  text!:string;

}

File: app/src/main/java/com/wild/corp/adhesion/models/resources/Groupe.java
package com.wild.corp.adhesion.models.resources;


import lombok.Data;

import java.util.List;

@Data
public class Groupe {


   private Boolean bold;
    private  Boolean checked;
    private   List<Horaire> horaires;

    private   Long id;
    private   Boolean indent;
    private   String    nom;
    private  Integer    ordre;
    private  String    text;
}
